app/components/dashboard/in_progress_row_component.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="row">
   <div class="col-4">
-    <%= collection_name %> &gt; <%= edit_work_link %>
+    <span title="<%=collection_name%>"><%= truncate(collection_name, length: 100, separator: ' ') %></span> &gt; <%= edit_work_link %>
```

Comment:
Please move this to a method in the class

Reply:
done